docs/segment.md

```diff
@@ -2,98 +2,188 @@
+
+You can now draw bounding boxes around the text regions you want to annotate by clicking and dragging a zone (green by default).
+
+![image: Drawing bounding boxes in the Segmenting panel of eScriptorium](img/segment/bounding_boxes_drawing.gif "Several bounding boxes are drawn around meaningful text regions.")
+
+You can modify a text region by clicking on it, and by dragging it to the dimensions you want.
+
+![image: Resizing a text region](img/segment/resizing_text_region.gif "")
+
+You can delete a text region by clicking on it, still in region mode, and by clicking on the delete button, or by pressing ++delete++.
+
+!!! note
+    Keep in mind that you can always press ++ctrl+z++ to undo any modification.
+
+### Labeling segmented text regions with an ontology
+
+eScriptorium allows to label segmented text regions with an ontology that you can customize.
+
+The ontology panel is one of the tabs you can access once you are working in a document. It is accessible with (`{base_url}/document/{document-id}/ontology/`). Text regions have a dedicated field named *region types*.
+
+![image: Ontology tab](img/segment/ontology_tab.png "Illustration of the ontology tab.")
+
+eScriptorium uses four region types by default:
+
+- Commentary
+
+- Illustration
+
+- Main
+
+- Title
+
+You can use them, or create your own. To deactivate default region types, you can deactivate them by unticking the ones you want to discard, and then by clicking on the update button.
+
+![image: Deactivating default text regions](img/segment/default_text_regions.gif "Modifying the text region ontology by deactivating default region types.")
+
+You can then create your own by writing the text region name you want into the dedicated field and then the + button. Don't forget to click on the update button to save your modifications!
+
+![image: Creating a custom ontology](img/segment/create_text_regions_ontology.gif "Adding region types to the text region ontology.")
+
+Then, once you are back to the segmentation panel, you can add a label to the bounding boxes you drew earlier by clicking on them, still in region mode, and then by clicking on the 'Set the type on all selected lines/regions' button, or by pressing ++t++. A dropdown menu should appear with the custom labels you created.
+
+![image: Labeling a text region ](img/segment/labeling_text_regions.gif "We click on a text region and label it with a label from the text region ontology.")
 
-![image: Screenshot of the 5 activated panels (Segmentation panel framed in red) ](img/segment/segment_panel.png "The 3d panel is used to manually edit the segmentation")
+!!! tip
+    You can change the color of a text region in eScriptorium in the **images** tab by clicking on **editor settings**. 
 
-## Overview of the segmentation panel
+!!! note
+    It is possible to create nested text regions, just draw another bounding box on top of the text regions you want to annotate!
 
-<!-- rewrite this section-->
+!!! note
```

Comment:
I think that note here is weird, it does not seem really relevant and you already said it normally at the beginning